docs/security/trust/content_trust.md

```diff
@@ -172,8 +172,8 @@ and push a tagged image. If this is the first time you have pushed an image
 using content trust on your system, the session looks like this:
 
 ```bash
-$ docker push docker/trusttest:latest
-The push refers to a repository [docker.io/docker/trusttest] (len: 1)
+$ docker push <username>/trusttest:testing
```

Comment:
Thanks for fixing this @mansinahar.  For consistency, could either we change this tag to `latest` here and in the above `build` step, or change the small note at the bottom of this section to refer to the `testing` tag (where it says that the tag cannot be omitted)?